x-pack/plugins/entity_manager/server/routes/entities/index.ts

```diff
@@ -10,11 +10,14 @@ import { deleteEntityDefinitionRoute } from './delete';
 import { getEntityDefinitionRoute } from './get';
 import { resetEntityDefinitionRoute } from './reset';
 import { updateEntityDefinitionRoute } from './update';
+import { searchEntitiesRoute, searchEntitiesPreviewRoute } from './search';
 
 export const entitiesRoutes = {
   ...createEntityDefinitionRoute,
   ...deleteEntityDefinitionRoute,
   ...getEntityDefinitionRoute,
   ...resetEntityDefinitionRoute,
   ...updateEntityDefinitionRoute,
+  ...searchEntitiesRoute,
+  ...searchEntitiesPreviewRoute,
```

Comment:
We agreed with @tommyers-elastic while you were on PTO to just put these under a v2 folder and add v2 to the route name for now.